### DISCUSSION TITLE
would like to add a new Python package for spike prime robot

### DISCUSSION THREAD

**Discussion by pecan-zq** [QuirkyCort/gears#121] [Q&A]
I would like to add a new Python package named "spike" so that my .py file can import it like 

`from spike import ColorSensor, DistanceSensor, MotorPair, PrimeHub`

However, no matter what I tried, it seems the simulator won't be able to find this package. It keeps giving me the error `"ImportError: No module named spike on line 8".` Can anyone help me on how to add a new Python package?

**Comment by QuirkyCort** [comment]
You need to add the new file to the "externalLibs" list in "public/js/skulpt.js". It's pretty easy to do, just refer to the existing listings.

The only part that may be a little confusing is the "?v=08e6bcd6" at the end of every filename; that's to prevent browsers from reusing old, cached versions of the file. You can simply exclude it, or put in a random value after the "=" (eg. "?v=random123").
